Домашнее задание № 1 третьего урока.

lenysi4ka25 opened the pull request:
Продолжаю осваивать язык java)))
Домашнее задание № 2 третьего урока

lenysi4ka25 committed:
Домашнее задание № 1 третьего урока.
Продолжаю осваивать язык java)))
Домашнее задание № 2 третьего урока

lenysi4ka25 committed:
Домашнее задание № 1 третьего урока.
Продолжаю осваивать язык java)))
Домашнее задание № 2 третьего урока

lenysi4ka25 committed:
Домашнее задание № 1 третьего урока.
Продолжаю осваивать язык java)))
Домашнее задание № 2 третьего урока.
Домашнее задание № 3 третьего урока.

lenysi4ka25 committed:
Домашнее задание № 1 третьего урока.
Продолжаю осваивать язык java)))
Домашнее задание № 2 третьего урока.
Домашнее задание № 3 третьего урока.
Домашнее задание № 4 третьего урока (абстрактный класс) - готово)))

mochaches commented on src/Train.java:
```diff
@@ -0,0 +1,103 @@
+    public Train(String marka, String model, int yearOfProduction, String countryOfProduction, String colour,
+                 int maxSpeed, int costTrip, String travelTime, String departureStations, String terminalStation, int numberWagons) {
+        super();
+        if (costTrip <= 0) {
```
логике в конструкторе быть не должно, для этого есть set-еры. И зачем вызывать пустой конструктор родителя?

mochaches commented on src/Transport.java:
```diff
@@ -1,13 +1,13 @@
-    private final String countryOfProduction;
-    private String colour;
-    private int maxSpeed;
+    protected String marka;
```
почему protected?

lenysi4ka25 committed:
Домашнее задание № 1 третьего урока.
Продолжаю осваивать язык java)))
Домашнее задание № 2 третьего урока.
Домашнее задание № 3 третьего урока.
Домашнее задание № 4 третьего урока (абстрактный класс) - готово)))
Внесла изминения согласно комментов

lenysi4ka25 committed:
Домашнее задание № 1 третьего урока.
Продолжаю осваивать язык java)))
Домашнее задание № 2 третьего урока.
Домашнее задание № 3 третьего урока.
Домашнее задание № 4 третьего урока (абстрактный класс) - готово)))
Внесла изминения согласно комментов
В классе Transport изменила модификаторы доступа!
